### DISCUSSION TITLE
Can ts-node have its own config file?

### DISCUSSION THREAD

**Discussion by stdedos** [TypeStrong/ts-node#2080] [Q&A]
I'd like that, instead of updating `tsconfig.json` to use `swc`, I could select `ts-node` to use it instead.

`tsconfig.json` is (usually) a production-committed artifact. ts-node fits the REPL role - which is more "dev preferance"-specific

**Comment by alexandre-embie** [comment]
yes just do ts-node --project tsconfig.dev.json myfile.ts to use a specific tsconfig.json

**Reply by stdedos** [reply]
Can it still inherit (but override) `tsconfig.json`?
